js/main.js

```diff
@@ -62,6 +64,68 @@ function renderResults(name, street, city, state, zip) {
   $info.textContent = city + ', ' + state + ', ' + zip;
   initialDiv.appendChild($info);
 
+  var $url = document.createElement('a');
+  if (url !== null) {
+    $url.setAttribute('href', url);
+    $url.textContent = 'Website: ' + url;
+  } else {
+    $url.textContent = 'No Website Available';
+  }
+  initialDiv.appendChild($url);
+  $url.className = 'view hidden';
+
+  function titleCase(string) {
+    string = string.toLowerCase().split(' ');
+    for (var i = 0; i < string.length; i++) {
+      string[i] = string[i].charAt(0).toUpperCase() + string[i].slice(1);
+    }
+    return string.join(' ');
+  }
```

Comment:
Why is this function definition inside of another function definition? You should move this to the global scope